TOOLS/lua/autoload.lua

```diff
@@ -49,16 +49,18 @@ function SetUnion (a,b)
 end
 
 EXTENSIONS_VIDEO = Set {
-    '3gp', 'avi', 'flv', 'm2ts', 'm4v', 'mkv', 'mp4', 'mpeg', 'mpg',
-    'ogv', 'rmvb', 'webm', 'wmv'
+    '3g2', '3gp', 'avi', 'flv', 'm2ts', 'm4v', 'mj2', 'mkv', 'mov',
+    'mp4', 'mpeg', 'mpg', 'ogv', 'rmvb', 'webm', 'wmv', 'y4m'
 }
 
 EXTENSIONS_AUDIO = Set {
-    'flac', 'm4a', 'mp3', 'ogg', 'ogm', 'opus', 'wav', 'wma'
+    'aiff', 'ape', 'au', 'flac', 'm4a', 'mka', 'mp3', 'oga', 'ogg',
+    'ogm', 'opus', 'spx', 'wav', 'wma'
 }
 
 EXTENSIONS_IMAGES = Set {
-    'bmp', 'gif', 'jpeg', 'jpg', 'png', 'svg', 'tif', 'tiff', 'webp'
+    'avif', 'bmp', 'gif', 'j2k', 'jp2', 'jpeg', 'jpg', 'jxl', 'png',
+    'svg', 'tga', 'tif', 'tiff', 'webp'
 }
 
 EXTENSIONS = Set {}
```

Comment:
seems fine but two things:
I doubt anyone actually has any `spx` or `svg` files laying around

Reply:
svg was already there, I can remove speex though

Reply:
do whatever you deem correct :shrug: